app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ]  Great job setting up devise for your app. However, there's one required configuration that's left out. It is required to configure confirmable module to handle the newly registered user to confirm his or her email before having access to to login. Kindly add that too. Great job so far guys. :+1: